### PR TITLE
Notify the share owner when another user downloads a shared file

### PR DESCRIPTION
**Pull request by oparoz**
Part 1 of the solution for #853
You will find part 2 at https://github.com/nextcloud/activity/pull/19

### CODE REVIEW

**Comment by oparoz** [comment]
@blizzz - This is the PR I was talking about where a user said downloads from people authenticated via LDAP were not tracked.

File: apps/files_sharing/lib/Activity.php [hunk 1]
```diff
@@ -264,7 +274,10 @@ protected function translateShort($text, IL10N $l, array $params) {
 			case self::SUBJECT_PUBLIC_SHARED_FOLDER_DOWNLOADED:
 			case self::SUBJECT_PUBLIC_SHARED_FILE_DOWNLOADED:
 				return (string) $l->t('Downloaded via public link');
-
+			case self::SUBJECT_SHARED_FOLDER_DOWNLOADED:
+			case self::SUBJECT_SHARED_FILE_DOWNLOADED:
+				return (string) $l->t('%1$s was downloaded by %2$s (%3$s)', $params);
```
Comment (nickvergessen) [inline apps/files_sharing/lib/Activity.php]: For short translation (sidebar) remove the file name:
`'Downloaded by %2$s (%3$s)'`

Comment (oparoz) [inline apps/files_sharing/lib/Activity.php]: Ah, good point :)

**Comment by MorrisJobke** [comment]
Tested and works 👍 

@nickvergessen Feel free to merge

**Comment by blizzz** [comment]
Tested with LDAP users, works :+1:

**Comment by MorrisJobke** [comment]
@nickvergessen Is this now ready for review?

**Comment by nickvergessen** [comment]
Replaced with this new app, so the feature can be disabled by default (not installing the app)
https://github.com/nextcloud/files_downloadactivity

**Pull request closed without merging**